cli/cmd/plugin/ui/web/tanzu-ui/node-server/user_data/logs/access.log

```diff
@@ -3441,3 +3441,7 @@
 2022-04-25T20:19:36.629Z ::ffff:127.0.0.1 - "GET /api/provider/aws/regions HTTP/1.1" 200 41 - 1.815 ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.88 Safari/537.36"
```

Comment:
Raymond, sorry would you mind adding this file to .gitignore please?

Reply:
will do it